README.md

```diff
@@ -1,4 +1,28 @@
 # Kotlin Wallet SDK
 
 Kotlin Wallet SDK is a library that allows developers to build wallet applications on the Stellar network faster. It
-utilizes [Java Stellar SDK](https://github.com/stellar/java-stellar-sdk) to communicate with a Stellar Horizon network.
\ No newline at end of file
+utilizes [Java Stellar SDK](https://github.com/stellar/java-stellar-sdk) to communicate with a Stellar Horizon network.
+
+## Wallet
+
+`class Wallet(horizonUrl: String, horizonPassphrase: String)`
+
+Example
+
+```kotlin
+// Using Horizon testnet configuration
+val wallet = Wallet("https://horizon-testnet.stellar.org", "Test SDF Network ; September 2015")
+```
+
+### `Wallet.create()`
+
+`fun create(): AccountKeypair(val publicKey: String, val secretKey: String)`
```

Comment:
The Java SDK's `Keypair.random()` does the same thing as this function.

Is the intention for the wallet sdk's functionality to overlap with the base Java SDK, or to be supplemental?

Reply:
The idea is to make it easier for anyone to use Kotlin Wallet SDK. `Wallet.create()` is just a wrapper for `Keypair.random()` but it will be more consistent with all the other SDK methods.